addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/KodiEventListener.java

```diff
@@ -30,10 +31,21 @@
         FastForward
     }
 
+    public enum KodiPlaylistState {
+        Add,
```

Comment:
Enums should be written in caps, because if I look quickly it seems that there is no reason to not comply in this case?

Reply:
Hmm, lets do that another time, good to get it wrapped up now :-)